_pkgdown.yml

```diff
@@ -11,7 +17,16 @@ template:
     twitter:
       creator: "@AtorusR"
       card: summary_large_image
-
+search:
+  exclude: ['news/index.html']
```

Comment:
Is it standard practice to exclude the news file? My thought is there would be useful info there if people needed to search it.

Reply:
I took that out of the admiral template.